src/index.html

```diff
@@ -10,23 +10,39 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search bar</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+    >
+      <label>
+        <input
+          class="input__search input__search--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="small"
+    >
+      <label>
+        <input
+          class="input__search input__search--small"
+          type="text"
+          data-qa="small"
```

Comment:
The `data-qa` attribute is duplicated in the second form's input element. It should have a unique value to differentiate it from the `data-qa` attribute used in the form element. Consider using a different value or removing it if not necessary.